scripts/rpm-prepare.sh

```diff
@@ -13,8 +13,8 @@ export ODO_RPM_VERSION=${ODO_VERSION//-}
 
 # Golang version variables, if you are bumping this, please contact redhat maintainers to ensure that internal
 # build systems can handle these versions
-export GOLANG_VERSION=${GOLANG_VERSION:-1.13}
```

Comment:
I'm afraid that this change won't be possible right now. Build root that we use on Brew doesn't include go 1.15 :-( cc: @mohammedzee1000

Reply:
Lets go ahead

Reply:
I have already requested upgrade in brew. Worst case I can override envs